utils/src/main/scala/za/co/absa/enceladus/utils/implicits/StringImplicits.scala

```diff
@@ -203,5 +195,47 @@ object StringImplicits {
     def coalesce(alternatives: String*): String = {
       alternatives.foldLeft(string)(_.nonEmpyOrElse(_))
     }
+
+
+    /**
+      * Splits the string around the provided delimiter, unless it's inside quotes
+      * from https://stackoverflow.com/questions/1757065/java-splitting-a-comma-separated-string-but-ignoring-commas-in-quotes
```

Comment:
Nice! I would probably have just done `string.split(",")` and then cried when commas in quotes were given 😆 .